contracts/token/RefundableToken.sol

```diff
@@ -0,0 +1,70 @@
+pragma solidity ^0.4.8;
+
+
+import './StandardToken.sol';
+import '../ownership/Ownable.sol';
+import '../SafeMath.sol';
+
+
+/**
+ * @title Refundable token
+ * 
+ * @dev Simple ERC20 Token example, with refundable token creation.
+ * @dev Issue: https://github.com/OpenZeppelin/zeppelin-solidity/issues/217
+ *
+ * @notice Your main contract should inherit form this contract and implement the canRefund() method.
+ * @notice Warning -> If you do not implement canRefund() this will be an abstract contract.
+*/
+
+
+contract RefundableToken is StandardToken, Ownable {
+  event Refund(address indexed to, uint amount);
+
+  uint public refundRate = 100;
```

Comment:
Does it make sense to add an initial `refundRate`? It's very specific to the token that inherits from this one and it might be better to leave it abstract somehow, to prevent errors.